apps/storybook.namekit.io/stories/Namekit/DisplayedPriceConversion.stories.tsx

```diff
@@ -0,0 +1,134 @@
+import { Currency, numberAsPrice } from "@namehash/ens-utils";
+import {
+  CurrencySymbol,
+  CurrencySymbology,
+  CurrencySymbolPosition,
+  DisplayedPriceConversion,
+  PriceDisplaySize,
+} from "@namehash/namekit-react";
+import type { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+
+const meta: Meta<typeof DisplayedPriceConversion> = {
```

Comment:
@FrancoAguzzi I don't understand what is causing all the issues and mistakes that are happening. Can you please help me understand what is going wrong and causing so many issues?

For example, all of the current stories for `DisplayedPriceConversion` totally fail to communicate the big idea for this UI component. Maybe that's because you don't understand what the big idea is yourself? If you don't know you need to be asking more questions. The big idea for this UI component is to handle the UX for displaying a price in multiple currencies. For example, a NFT might be priced at 1 ETH, but the user might want to also see what that price represents in USD.

If you understand the big idea for this UI component, then why would the stories that are defined here all be so useless and confusing? For example, it's useless to show a conversion from ETH to ETH or from ETH to WETH. That's silly. I shouldn't need to catch these kinds of mistakes myself. You're making our UI components look pointless and confusing by failing to communicate the big idea to the audience who will look at these stories.

The next big issue is I continue to see a clear lack of you doing your own testing and quality control. For example, check the screenshot below:

![CleanShot 2024-08-30 at 13 58 32](https://github.com/user-attachments/assets/70e74f0a-3ea9-403e-99d0-2d1521314d3a)

Before you submit a PR for review, you should always first take on the responsibility of reviewing everything that PR is changing and verifying that all outputs of that changed code is correct. How do you think my process works where I review a PR and find so many issues to fix? I'm following a very simple process to find all of these problems. You should be following this process yourself and finding + fixing these issues before you ask me to review your work.

Reply:
@lightwalker-eth https://github.com/namehash/namekit/pull/361#discussion_r1737312089

Reply:
Hi @lightwalker-eth !

About: 
1. Examples used: I like how WETH is displayed visually but you are correct :: WETH <> ETH conversions are not very intelligent!! I understand how this is well known in the ecosystem, the goal was to display this component beautifully and I am happy to exchange WETH to DAI to continue doing so, in a more intelligent way;
2. Duplicate of Tooltip: [I've mentioned](https://github.com/namehash/namekit/pull/361#discussion_r1737312089) how it would be nice to add a custom `describeCurrencyInATooltip` prop in `DisplayedPrice`-- we have previously decided to not add `CurrencySymbol` logics to `DisplayedPrice` for separation of ideas but now we see how this is needed in order to remove the default `symbol` custom prop (which aims to remove the `Tooltip` of `CurrencySymbol` whenever the `DisplayedPrice` is rendered inside `DisplayedPriceConversion`)